pythran/config.py

```diff
@@ -53,6 +53,19 @@ def init_cfg(sys_file, platform_file, user_file):
         cfgp.read([required])
     cfgp.read([user_config_path])
 
+    if config_args is not None:
+        # Override the config options with those provided on the command line e.g. compiler.blas=pythran-openblas
+        for arg in config_args:
+            try:
+                lhs, rhs = arg.split('=')
```

Comment:
you should specify `maxsplit=1` here in case we pass `compiler.cxxflags=-fsanitize=address` or similar stuff